concrete/src/Job/Factory.php

```diff
@@ -0,0 +1,240 @@
+<?php
+namespace Concrete\Core\Job;
+
+use Concrete\Core\Application\Application;
+use Concrete\Core\Package\PackageList;
+
+class Factory
+{
+    /** @var \Concrete\Core\Application\Application */
+    private $app;
+
+    public function __construct(Application $app)
+    {
+        $this->app = $app;
+    }
+
+    /**
+     * @param bool $scheduledOnly
+     *
+     * @return Job[]
+     */
+    public function getList($scheduledOnly = false)
```

Comment:
Can this be `$factory->installed()`, and can we add a method `$factory->scheduled()` to replace this overloaded `$scheduledOnly = false` functionality?